### PR TITLE
Jdf/bigcz style

### PR DESCRIPTION
**Pull request by jfrankl**
## Overview

Style cleanup for BiG CZ application.

Connects #1938 

### Demo

image

image

### Notes

* I removed hyperlinks in the search results text. Users will eventually be clicking on those result boxes, so we should keep them simple. We can expose those links in the detailed view if they're important.
* The pagination should ideally be fixed to the bottom of the page. This is going to take some collaboration with a developer to pull off. 

## Testing Instructions

 * Go through the BiG CZ standard workflow, make sure everything looks okay.

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now.

**Comment by mmcfarland** [comment]
The date range filters don't appear in the screenshots posted - was this based on the most recent UI version?

**Comment by rajadain** [comment]
Also, would it be possible to left align the overflowing lines to the first line? Like this:

image

File: src/mmw/sass/pages/_data-catalog.scss [hunk 1]
```diff
@@ -46,9 +69,9 @@
         }
         .data-catalog-search-icon {
             color: $ui-grey;
-            position: relative;
```
Comment (mmcfarland) [inline src/mmw/sass/pages/_data-catalog.scss]: This was previously set to `relative` to account for the changing height when toggling the date filters.  Using `absolute` keeps it fixed even when the search box itself is pushed down.

**Comment by jfrankl** [comment]
@rajadain I believe I've made all of the fixes you requested. Take a look:

image

File: src/mmw/sass/pages/_data-catalog.scss (hunk 1 above)
Comment (jfrankl) [inline src/mmw/sass/pages/_data-catalog.scss]: I'm not sure what this means, Matt. I'm going to need some developer support to get the pagination fixed in the correct way. Previously, the pagination didn't move, which was good, but there were other issues with the sidebar layout. I fixed the other issues but I need help to get the pagination working right.

**Comment by rajadain** [comment]
Taking another look.

**Comment by rajadain** [comment]
We seem to have hidden the many sample method fields behind a an ellipsis:

image

Since we have a recommendation of possibly dropping the field entirely (see: https://github.com/WikiWatershed/model-my-watershed/issues/2096#issuecomment-318682012) I think this is fine for now.

File: src/mmw/sass/pages/_data-catalog.scss (hunk 1 above)
Comment (rajadain) [inline src/mmw/sass/pages/_data-catalog.scss]: We have a card to fix pagination #2091. The date filter opens and closes correctly, pushing things down and up as it should. Don't think this is an issue here.

**Comment by rajadain** [comment]
I'm going to merge these in before they get too far behind `develop` and have conflicts to deal with.

**Comment by rajadain** [comment]
Sorry, had forgotten to assign you @jfrankl. Is this good to merge?

**Comment by jfrankl** [comment]
Yes please go ahead.

**Comment by rajadain** [comment]
Thanks for working on this!

**Pull request merged**